.travis.yml

```diff
@@ -11,17 +11,18 @@ services:
 cache: npm
 jobs:
   include:
-    - stage: test
-      script:
-        # this will only run lint / tests against modified files but it is unstable
-        # - echo $TRAVIS_COMMIT_RANGE | sed 's/\.\.\./../' | xargs git diff-tree --no-commit-id --name-only -r | xargs echo -n
-        - npm run lint || travis_terminate 1
-        - npm test || travis_terminate 1
-    - stage: deploy
-      script:
-        - NODE_ENV=production REACT_APP_COMMIT_HASH=$TRAVIS_COMMIT npm run build
-        - docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD" quay.io
-        - bash docker.sh
+  - stage: test
+    script:
+    # this will only run lint / tests against modified files but it is unstable
+    # - echo $TRAVIS_COMMIT_RANGE | sed 's/\.\.\./../' | xargs git diff-tree --no-commit-id --name-only -r | xargs echo -n
+    - npm run lint -- --max-warnings=0 || travis_terminate 1
```

Comment:
Looks like something changed with the build and the `--max-warnings` option isn't accepted anymore. Probably better to update this part to match what's in develop now (I think this option was there way back when the PR was first opened and then removed later).